front/src/routes/spotRequest.js

```diff
@@ -20,4 +20,23 @@ async function saveSpot(spotName, x, y, picture, spotType, userId, comment, scor
     }
 }
 
-export {saveSpot};
\ No newline at end of file
+async function getSpot(spotId, spotName, spotType, userId){//xMax, xMin, yMax, yMin
+    // const encode = encodeURI(userId)
+    const url = serverIP + '/spot/getSpot?' + "spotId=" + spotId + "&spotName=" + spotName  + "&spotType=" + spotType + "&userId" + userId;//encode //"&xMax=" + xMax + "&xMin=" + xMin + "&yMax" + yMax + "&yMin" + yMin
+    try{
+        let response = await fetch(url,);
+        let responseJson = await response.json();
+        //Should probably use status code instead of this
+        if (responseJson.success){
+            return [responseJson.data, responseJson.review];
```

Comment:
I had to go check in userSQL what is data. If we are only sending one type of data it is usually clear what object we are receiving but since we are receiving two types of data here, we should rename data to spot or spots so that we know that we are getting spots and reviews. We should probably change the attribute name in the spotController and the spotSQL